hummingbot/client/config/client_config_map.py

```diff
@@ -906,12 +899,13 @@ class ClientConfigMap(BaseClientModel):
                      "\ndefault host to only use localhost"
                      "\nPort need to match the final installation port for Gateway"),
     )
-    certs: CertsConfigMap = Field(
-        default=CertsConfigMap(),
-        description=("Certs Configurations"
-                     "\ndefault: use the client generated certs"
-                     "\nPort need to match the certifactes for Gateway"),
+    certs: Path = Field(
+        default=DEFAULT_GATEWAY_CERTS_PATH,
+        client_data=ClientFieldData(
+            prompt=lambda cm: f"Where would you like to save your logs? (default '{DEFAULT_GATEWAY_CERTS_PATH}')",
```

Comment:
Change question to "Where would you like to save certificates that connect your bot to Gateway?"